modules/core/src/test/java/org/apache/ignite/internal/metric/JmxMetricExporterSpiTest.java

```diff
@@ -144,4 +210,125 @@ public void testFilterAndExport() throws Exception {
 
         assertEquals(44L, bean2.getAttribute("test3"));
     }
+
+    /** */
+    @Test
+    public void testCachesList() throws Exception {
+        Set<String> cacheNames = new HashSet<>(Arrays.asList("cache-1", "cache-2"));
+
+        for (String name : cacheNames)
+            ignite.createCache(name);
+
+        TabularDataSupport data = monitoringList("caches");
+
+        assertEquals(3, data.size());
```

Comment:
I think something like `ignite.context().cache().cacheDescriptors().size()` will be better then just a constant.